src/protagonist/API/Features/Image/Ingest/DeliveryChannelProcessor.cs

```diff
@@ -73,20 +72,20 @@ private bool DeliveryChannelsRequireReprocessing(Asset originalAsset, DeliveryCh
         return false;
     }
 
-    private async Task<bool> SetImageDeliveryChannels(Asset asset, DeliveryChannelsBeforeProcessing[] deliveryChannelsBeforeProcessing, bool isUpdate)
+    private async Task<bool> SetImageDeliveryChannels(Asset asset, DeliveryChannelsBeforeProcessing[]? deliveryChannelsBeforeProcessing, bool isUpdate)

```

Comment:
Line 95 in this class is giving a nullable warning - by this stage do we need to have `null` passed down? Would it simplify to `?? Array.Empty` and treat null + empty as the same without worrying?